src/lib/utils/arweaveUtils.ts

```diff
@@ -1,47 +1,54 @@
 import * as B64js from "base64-js";
 
-export type Base64UrlString = string;
-
-export function concatBuffers(
-  buffers: Uint8Array[] | ArrayBuffer[],
-): Uint8Array {
-  let total_length = 0;
-
-  for (let i = 0; i < buffers.length; i++) {
-    total_length += buffers[i].byteLength;
-  }
-
-  let temp = new Uint8Array(total_length);
-  let offset = 0;
+// BINARY DATA TYPES:
 
-  temp.set(new Uint8Array(buffers[0]), offset);
-  offset += buffers[0].byteLength;
-
-  for (let i = 1; i < buffers.length; i++) {
-    temp.set(new Uint8Array(buffers[i]), offset);
-    offset += buffers[i].byteLength;
-  }
+export type Base64UrlString = string;
 
-  return temp;
+export type TypedArray =
+  | Int8Array
+  | Uint8Array
+  | Uint8ClampedArray
+  | Int16Array
+  | Uint16Array
+  | Int32Array
+  | Uint32Array
+  // | Float16Array
```

Comment:
TS says this one doesn't exist. I'll try again in a new PR once I update the dependencies and see if it is there in the latest version.